lib/matplotlib/colors.py

```diff
@@ -529,7 +529,18 @@ def __call__(self, X, alpha=None, bytes=False):
         if vtype == 'scalar':
             rgba = tuple(rgba[0, :])
         return rgba
-
+    
+    def __copy__(self):
+        """Create new object with the same class and update attributes
+	If object is initialized, copy the elements of _lut list
+	"""
+	cls = self.__class__
```

Comment:
Get a color map, use it (`cm([-1, 0, .5, 1, np.nan, np.inf])`, copy it, use `set_bad`, `set_over`, `set_under` on one or both of them, use them both and check that the colors come back as expected.